pkg/utils/types.go

```diff
@@ -0,0 +1,6 @@
+package utils
+
+// NewBool returns a pointer to a new initialized bool value.
+func NewBool(value bool) *bool {
```

Comment:
You can use https://pkg.go.dev/github.com/bitrise-io/go-utils/pointers#NewBoolPtr instead of adding a new function

Reply:
Okayh, I will back off to that one.